<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PodcastFilenameLayoutConfigWidget</class>
 <widget class="QWidget" name="PodcastFilenameLayoutConfigWidget">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>110</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
    <horstretch>250</horstretch>
    <verstretch>100</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Podcast Filename Layout Configuration</string>
  </property>
  <widget class="QRadioButton" name="m_filenameLayoutDefault">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>85</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Specified by podcast channel</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_filenameLayoutCustom">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>115</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>custom</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_filenameLayoutText">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
     <horstretch>200</horstretch>
     <verstretch>20</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>20</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>200</width>
     <height>20</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Available fields : %artist%,%title%,%genre%,%year%,%composer%,%pubdate%,%number%,%album%</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_filenameLayoutCustom</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_filenameLayoutText</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
